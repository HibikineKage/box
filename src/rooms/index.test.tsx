import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Rooms from './index';

test('rendering', () => {
  const tree = renderer.create(<Rooms />);
  expect(tree.toJSON()).toMatchSnapshot();
});
