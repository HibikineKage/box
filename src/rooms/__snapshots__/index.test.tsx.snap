// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<div>
  <h1>
    Lobby
  </h1>
  <section>
    <h2>
      Room List
    </h2>
    <p>
      <button
        onClick={[Function]}
      >
        Test Room 1
      </button>
    </p>
    <p>
      <button
        onClick={[Function]}
      >
        Test Room 2
      </button>
    </p>
    <p>
      <button
        onClick={[Function]}
      >
        Test Room
      </button>
    </p>
  </section>
  <div>
    <button
      onClick={[Function]}
    >
      部屋を追加する
    </button>
    <input
      onChange={[Function]}
      type="text"
      value=""
    />
  </div>
  <div>
    <button
      onClick={[Function]}
    >
      CPU Battle
    </button>
  </div>
</div>
`;
